api/db-cdr/generate-cdr/make-cloudsql-data.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# This generates cloudsql data from the big query cdr dataset created
+# by make-bq-data.sh
+# note  the account must be authorized to perform gcloud and bq operations
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+USAGE="./generate-clousql-cdr/make-bq-data.sh --project <PROJECT> --account <ACCOUNT> --cdr-version=YYYYMMDD"
+while [ $# -gt 0 ]; do
+  echo "1 is $1"
+  case "$1" in
+    --account) ACCOUNT=$2; shift 2;;
+    --project) PROJECT=$2; shift 2;;
+    --cdr-version) CDR_VERSION=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+if [ -z "${ACCOUNT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${PROJECT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${CDR_VERSION}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+#Check cdr_version is of form YYYYMMDD
+if [[ $CDR_VERSION =~ ^[0-9]{4}(0[1-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])$ ]]; then
+    echo "New CDR VERSION will be $CDR_VERSION"
+  else
+    echo "CDR Version doesn't match required format YYYYMMDD"
+    echo "Usage: $USAGE"
+    exit 1
+fi
+
+CREDS_ACCOUNT=${ACCOUNT}
+
+# Variables
+# TODO .... all this is just old code for reference
```

Comment:
If you're not planning on keeping this file... you could remove it from this PR, make a copy of it locally if you want. You still have the old PRs for reference.

Reply:
yeah removed.